Dockerfile

```diff
@@ -1,7 +1,5 @@
-FROM quay.io/higginsd/resource-downloader
+FROM docker.io/centos:centos7
 
 RUN yum install -y epel-release && yum install -y qemu-img jq && yum clean all
```

Comment:
The resource-downloader base image had a yum update to start from an uptodate image, can we also add that here?
RUN yum update -y && yum clean all

Reply:
Ah good point, now done